code/index.html

```diff
@@ -16,25 +16,33 @@
 
       <img
         class="guess-who-icon"
-        src="images/guess-who-bubble.png"
+        src="./images/guess-who-bubble.png"
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1 class="find-out-title">Does the person have</h1>
 
       <select id="questions">
         <optgroup label="hair color">
+          <option selected disabled default>---</option>
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
+          <option value="orange">orange hair</option>
+          <option value="grey">grey hair</option>
+          <option value="black">black hair</option>
+          <option value="purple">purple hair</option>
+          <option value="white">white hair</option>
+          <option value="hidden">hidden hair</option>
         </optgroup>
         <optgroup label="eye color">
+          <option value="blue">blue eyes</option>
+          <option value="brown">brown eyes</option>
+          <option value="green">green eyes</option>
           <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
         </optgroup>
         <optgroup label="accessories">
           <option value="glasses">glasses</option>
-          <!-- Add the other accessory option here. (hat)  -->
+          <option value="hat">hat</option>
         </optgroup>
         <optgroup label="other">
           <option value="smoker">a smoking habit</option>
```

Comment:
Maybe it would be a nice think to generate long lists of options with your Javascript? It is nothing wrong, with this approach, but It makes you type a lot 😸